app/install-packages.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# Install poetry
```

Comment:
This comment is not worth its line.

A comment explaining why poetry is installed a second time later in the file would be more helpful ^^

Reply:
`poetry install` installs the dependencies, not poetry itself.